tools/travis/backend

```diff
@@ -24,6 +24,7 @@ set -x
 #./tools/test-run-dev  # Disabled in CI because flaky.
 #./tools/test-queue-worker-reload  # Disabled in CI because flaky.
 
+./tools/test-slack-importer
```

Comment:
Maybe this should be run only at PR's that amend the Slack import code. @rheaparekh how long does the test usually last?

Reply:
The test here (./tools/test-locked-requirements + ./tools/test-slack-importer) took 9s.

Reply:
@rht I think it is better if the test would run only at PR's that amend the Slack import code. Can you guide me with this? I was unable to find a documentation.